cmd/runitor/main.go

```diff
@@ -71,7 +71,7 @@ func main() {
 	}
 
 	if *apiURL == internal.DefaultBaseURL {
-		v, ok := os.LookupEnv("CHECK_API_URL")
+		v, ok := os.LookupEnv("HC_API_URL")
 		if ok && len(v) > 0 {
```

Comment:
And handle it with:
```
if !urlFromArgs {
  if v, ok := os.LookupEnv("HC_API_URL"); ok && len(v) > 0 {
    apiUrl = &v   
  }
}
```

I know a bunch of users use runitor in containers they build with Docker or Podman. They would define the environment variable in the container file (nee Dockerfile) which will end up setting the variable to an empty value but still set if they forget to pass it when launching the container.

Simulated:
```
% cat env.go
package main
import (
  "fmt"
  "os"
)
func main() {
  v, ok := os.LookupEnv("FOO")
  fmt.Printf("v='%s' ok=%v\n", v, ok)
}

~% env FOO= go run env.go
v='' ok=true
```

In this case, we can either error out or ignore the empty env var and go with the default value that's set by `flag` description. I'd opt for the latter, hence the implementation above.